src/lib.rs

```diff
@@ -1,5 +1,11 @@
-use core::panic;
-use std::cmp;
+pub mod apply_diff;
+pub use  crate::apply_diff::{apply_diff};
+pub mod hamming;
+pub use  crate::hamming::{HammingDistance};
+pub mod levenshtein;
+pub use crate::levenshtein::{LevenshteinDistance};
```

Comment:
Can you make the spacing consistent here? So only 1 space in-between the pub use and the namespaced component, like `pub use crate::...` instead of `pub use  crate::...`.

Can you also remove the `pub` keyword before the modules, so it's `mod apply_diff` instead of `pub mod apply_diff`? By making the module public, it acts as a second nested namespace/module, but there would only be 1 thing in each module. Since there's also one thing, there shouldn't be brackets, so it should look like:
```rs
pub use crate::apply_diff::apply_diff; // this
pub use crate::apply_diff::{apply_diff}; // instead of this
```

Right now since the crate is small, everything should just be available from the root.